eth/protocols/eth/handler.go

```diff
@@ -179,334 +235,15 @@ func handleMessage(backend Backend, peer *Peer) error {
 	}
 	defer msg.Discard()
 
-	// Handle the message depending on its contents
-	switch {
-	case msg.Code == StatusMsg:
-		// Status messages should never arrive after the handshake
-		return fmt.Errorf("%w: uncontrolled status message", errExtraStatusMsg)
```

Comment:
Note for reviewer: This explicit error has been removed, and now this case would just fall through and error as an unhandled message